docs/UserGuide.md

```diff
@@ -103,18 +119,73 @@ Beyond individual products, you can also:
+</div>
+
+5. Now we would like to add in an upcoming meeting with our client 'Bernice Yu' (index 2 in our sample data). We can do that by doing `addMeeting i/2 d/27102022 st/1400 et/1600 dn/Review plan` (You can replace `d/27102022` with today's date in the DDMMYYYY format)! When we hit enter, *MyInsuRec* will show us the list of meetings, including the one we just added. The image below shows *MyInsuRec* after adding the meeting with Bernice.
+
+![AddMeetingBernice](images/quick-start/AddMeetingBernice.png)
+
+6. To view the meeting (with index number of 1) in greater detail, we can call `viewMeeting i/1`. This brings up a more detailed view of the meeting, with information such as description and Bernice's contact number. The image below shows *MyInsuRec* after using the `viewMeeting` command.
+
+![ViewMeetingBernice](images/quick-start/ViewMeetingBernice.png)
+
+7. Now, to go back to the list of clients, simply type in `listClient`. This brings us back to the list of clients.
+
+8. We would like to define and add more products! We can do so via `addProduct pd/MyInsureCare`. This adds a product 'MyInsureCare' and also brings us to the list of product with the newly added product. The image below shows *MyInsuRec* adding the product.
+
+![AddProductMyInsureCare](images/quick-start/AddProductMyInsureCare.png)
+
+9. Now, we want to add a client named who has already purchased 'MyInsureCare'. To do that, we can once again type in `addClient n/Ng Jun Yi p/81230987 pd/MyInsureCare` to add a client and indicate that he has bought 'MyInsureCare'. The image below shows *MyInsuRec* after adding the client with the product.
+
+![AddClientJunYiWithProduct](images/quick-start/AddClientJunYiWithProduct.png)
+
+<div markdown="block" class="alert alert-warning">:exclamation: **Caution:**
+Only products added already via `addProduct` can be used! This is to help ensure the cleanliness of *MyInsuRec*. So, add your product via `addProduct` before using it to add a client with that product! See [addProduct](#331-adding-a-product-addproduct).
+</div>
+
+10. We want to delete the client 'Alex Yeoh' from our records! To do that, type in `delClient i/1`. The image below shows *MyInsuRec* after deleting the client.
+
+![DelClientAlex](images/quick-start/DelClientAlex.png)
+
+And there we have it! We have just gone through the basic bookkeeping features of *MyInsuRec*. Beyond that, we also have features such as getting a quick glance of all your upcoming meetings, filtering clients by products. To find out and learn more about these features, head to the [Features section](#3-features) where all the commands and their details can be found.
 
-    1. `viewClient`: View a particular client
-    2. `addClient`: Add a client
-    3. `delClient`: Delete client
+To get started with an clean state of *MyInsuRec*, type in the command `clear`. This removes all the sample data from *MyInsuRec*. 
```

Comment:
Could probably show a screenshot of how it looks like after using the clear command

Reply:
hmm, im not sure if it is ideal because there's already many screenshots in this section and we are no longer in the steps + `clear` seems pretty clear to use. please let me know your POV of this!

Reply:
Or maybe u can add a one liner to say

"Dont be shock if all the information go missing" sth liddat. just to make them anticipate the change